src/OrbitGl/TimeGraphLayout.cpp

```diff
@@ -85,6 +90,11 @@ bool TimeGraphLayout::DrawProperties() {
   return needs_redraw;
 }
 
+float TimeGraphLayout::GetCollapseButtonSize(int indentation_level) const {
+  return collapse_button_size_ -
+      collapse_button_decrease_per_indentation_ * static_cast<float>(indentation_level);
+}
```

Comment:
I didn't like kMaxIndentationLevel, but you are right that I should force a minimum. Instead of writing them again, I will add kMinimumButtonSize, what do you think?

Reply:
I think we can have a global constant`kMaxIndentationLevel` such that it can be used in multiple places where  indentation level affects the size. For example, the track font size and the button size.

Reply:
Again, I don't like kMaxIndentationLevel unless we are really limiting the indentation level. I rather would prefer to have minimumButtonSize and minimumTextSize or limiting the indentation_level in DrawContext because it's only related to drawing.